packages/react-storage/src/components/StorageManager/StorageManager.tsx

```diff
@@ -93,21 +94,24 @@ function StorageManagerBase(
   React.useImperativeHandle(ref, () => ({ clearFiles }));
 
   const dropZoneProps = useDropZone({
```

Comment:
Can we just pass `onDropComplete` and `acceptedFileTypes` directly to the `DropZone`?

Reply:
We can, but wouldn't that be a breaking change for the StorageManager DropZone component slot?

Reply:
Potentially, but in the same way that changing the underlying DZ could also be considered a breaking change